sdk/eventhubs/pom.service.xml

```diff
@@ -36,10 +36,10 @@
         <module>../core/azure-core-http-netty</module>
         <module>../identity/azure-identity</module>
         <module>../core/azure-core-amqp</module>
-        <module>../storage/azure-storage-common</module>
-        <module>../storage/azure-storage-blob</module>
+<!--        <module>../storage/azure-storage-common</module>-->
```

Comment:
These should be uncommented.

Reply:
Oh shoot. I actually committed that. good catch.